node/connection.js

```diff
@@ -305,17 +314,17 @@ TChannelConnection.prototype.onCallResponse = function onCallResponse(res) {
         }
 
         called = true;
-
-        self.ops.popOutReq(res.id, {
-            responseId: res.id,
-            code: res.code,
-            arg1: Buffer.isBuffer(res.arg1) ?
-                String(res.arg1) : 'streamed-arg1',
-            info: 'got call response for unknown id'
-        });
+        self.popOutReq(res);
```

Comment:
the called boolean is O(n) call response; the conneciton is O(n) connection so it wont work.